### PR TITLE
Feat/63/cors

### PR DESCRIPTION
**Pull request by nandagopalan**
Added Cors Support

### CODE REVIEW

File: turbo/filters/cors.go [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+const (
+	AllowCredentials             = "Access-Control-Allow-Credentials"
+	AllowExposeHeaders           = "Access-Control-Expose-Headers"
+	AllowHeadersHeader           = "Access-Control-Allow-Headers"
+	AllowMethodsHeader           = "Access-Control-Allow-Methods"
+	AllowOriginHeader            = "Access-Control-Allow-Origin"
+	MaxAgeHeader                 = "Access-Control-Max-Age"
+	ExposeHeaders                = "Access-Control-Expose-Headers"
+	AccessControlReqHeaders      = "Access-Control-Request-Headers"
+	AccessConrtolReqMethodHdr    = "Access-Control-Request-Method"
```
Comment (neo7337) [inline turbo/filters/cors.go:21]: typo fix

File: turbo/filters/cors.go [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+	}
+
+	// Check if the origin is originAllowed
+	originAllowed, origin := cf.isOriginAllowed(reqOrigin)
+	if !originAllowed {
+		return
+	}
+
+	// Check if the method is allowed.
+	if originAllowed := cf.isMethodAllowed(r.Header.Get(AccessConrtolReqMethodHdr)); !originAllowed {
```
Comment (neo7337) [inline turbo/filters/cors.go:182]: methodAllowed instead of originAllowed

File: turbo/filters/cors.go [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+// HandleActualRequest handles the actual request
+func (cf *CorsFilter) HandleActualRequest(w http.ResponseWriter, r *http.Request) {
+	reqOrigin := r.Header.Get(OriginHeader)
+	// Check if the origin is originAllowed
+	originAllowed, origin := cf.isOriginAllowed(reqOrigin)
+	if !originAllowed {
+		return
+	}
+	// check if the method is allowed
+	if originAllowed := cf.isMethodAllowed(r.Method); !originAllowed {
```
Comment (neo7337) [inline turbo/filters/cors.go:222]: methodAllowed instead of originAllowed

**Review by neo7337 — APPROVED** [review]
approved

**Comment by nandagopalan** [comment]
Closes #66

**Pull request merged**